apps/watch/src/components/VideoGrid/VideoGrid.spec.tsx

```diff
@@ -1,26 +1,114 @@
-import { fireEvent, render } from '@testing-library/react'
-
+import { fireEvent, render, screen } from '@testing-library/react'
+import { useAlgoliaVideos } from '../../libs/algolia/useAlgoliaVideos'
+import type { AlgoliaVideos } from '../../libs/algolia/useAlgoliaVideos'
 import { videos } from '../Videos/__generated__/testData'
-
 import { VideoGrid } from './VideoGrid'
 
+jest.mock('react-instantsearch')
+jest.mock('../../libs/algolia/useAlgoliaVideos')
+
+const mockedUseAlgoliaVideos = useAlgoliaVideos as jest.MockedFunction<
+  typeof useAlgoliaVideos
+>
+
 describe('VideoGrid', () => {
-  it('should request more videos', async () => {
-    const onLoadMore = jest.fn()
-    const { getByText } = render(
-      <VideoGrid
-        videos={videos}
-        loading={false}
-        hasNextPage
-        onLoadMore={onLoadMore}
-      />
-    )
-    fireEvent.click(getByText('Load More'))
-    expect(onLoadMore).toHaveBeenCalled()
+  const algoliaVideos = [
+    {
+      videoId: 'videoId',
+      title: [
+        {
+          value: 'title1'
+        }
+      ],
+      description: ['description'],
+      duration: 10994,
+      languageId: '529',
+      subtitles: [],
+      slug: 'video-slug/english',
+      label: 'featureFilm',
+      image:
+        'https://d1wl257kev7hsz.cloudfront.net/cinematics/2_GOJ-0-0.mobileCinematicHigh.jpg',
+      imageAlt: 'Life of Jesus (Gospel of John)',
+      childrenCount: 49,
+      objectID: '2_529-GOJ-0-0'
+    }
+  ] as unknown as AlgoliaVideos[]
+  describe('Core Videos', () => {
+    it('should render core videos', () => {
+      mockedUseAlgoliaVideos.mockReturnValue({
+        hits: algoliaVideos,
+        showMore: jest.fn(),
+        isLastPage: false
+      })
+
+      render(<VideoGrid videos={videos} />)
+
+      expect(
+        screen.getByRole('heading', { level: 3, name: 'JESUS' })
+      ).toBeInTheDocument()
+      expect(
+        screen.getByRole('heading', {
+          level: 3,
+          name: 'Life of Jesus (Gospel of John)'
+        })
+      ).toBeInTheDocument()
+    })
   })
 
-  it('should render correct number of videos', () => {
-    const { getAllByLabelText } = render(<VideoGrid videos={videos} />)
-    expect(getAllByLabelText('VideoCard')).toHaveLength(videos.length)
+  describe('Algolia Videos', () => {
+    it('should render correct number of videos', async () => {
```

Comment:
```suggestion
    it('should render correct number of videos', async () => 
```

This test doesn't look like it tests for what is described in the it statement? says 'it should render correct number of videos, but just tests that title1 is on the screen